iothub_client/tests/iothubtransport_mqtt_common_ut/iothubtransport_mqtt_common_ut.c

```diff
@@ -2169,6 +2169,49 @@ TEST_FUNCTION(IoTHubTransport_MQTT_Common_SetOption_value_NULL_fail)
     IoTHubTransport_MQTT_Common_Destroy(handle);
 }
 
+TEST_FUNCTION(IoTHubTransport_MQTT_Common_SetOption_SAS_TOKEN_LIFETIME_NULL_fail)
```

Comment:
>IoTHubTransport_MQTT_Common_SetOption_ [](start = 14, length = 38)

Can we also please update the setoption comments in _LL with the option key/value?

Reply:
Sounds good.